cortx-motr.spec.in

```diff
@@ -52,15 +53,15 @@ Summary:        CORTX Motr
 Group:          System Environment/Base
 License:        Seagate
 Source:         %{name}-%{version}.tar.gz
-BuildArch:      x86_64
+BuildArch:      aarch64
```

Comment:
All these must be defined as conditional.
It should depend on the arch.
Please use "%if" to determine the target arch.